test/lazy-loading.html

```diff
@@ -852,6 +852,20 @@
               const pages = spyDataProvider.getCalls().map(call => call.args[0].page);
               expect(pages).to.contain(1);
             });
+
+            it('should set value if not opened nor focused', () => {
+              const dataProvider = (params, callback) => {
+                comboBox.blur();
```

Comment:
Nit: can we move `blurring` out of the `dataProvider`? I guess it's quite a rare case to `blur` the field from DP.

Reply:
The intention is that the field is focused when  `dataProvider()` is called, but is not focused when the results return and `callback()` is called. The tests fail otherwise

Reply:
I understand the intention of the test. I do mean that either the description of the test can be updated (i.e. `if not opened nor focused ` + **anymore**) or blur can be called outside the dp or the comment can be added to the line explaining why it is there 😅